README.md

```diff
@@ -114,3 +114,14 @@ NAME       STATUS    AGE
 10.0.0.4   Ready     20s
 
 ```
+
+## Customization
+
+User have possibility to execute custom scripts on master before and after provision process.
+
+Scripts must be stored in directory: /usr/local/scripts/ with following names:
+
+- pre-tasks.sh - executed before provision
+- post-tasks.sh - executed after provision
+
+User must add those scripts to the image with correct rights: rw-r-r
```

Comment:
If we create empty files by default we don't need this step. If we don't have them, then I would have two directories for post and pre and run every script located in these directories instead of keeping strict rules  for names.